src/modules/dashboard/components/ExpenditurePage/Batch/Batch.tsx

```diff
@@ -0,0 +1,216 @@
+import React, { useState } from 'react';
+import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
+import { useField } from 'formik';
+import classNames from 'classnames';
+import { isEmpty, isNil } from 'lodash';
+
+import { FormSection } from '~core/Fields';
+import { Colony } from '~data/index';
+import TokenIcon from '~dashboard/HookedTokenIcon';
+import Button from '~core/Button';
+import { useDialog } from '~core/Dialog';
+import IconTooltip from '~core/IconTooltip';
+
+import CSVUploader from './CSVUploader';
+import PreviewDialog from './PreviewDialog';
+import { useCalculateBatchPayment } from './hooks';
+import DownloadTemplate from './DownloadTemplate';
+import styles from './Batch.css';
+
+export const MSG = defineMessages({
+  batch: {
+    id: 'dashboard.ExpenditurePage.Batch.batch.',
+    defaultMessage: 'Batch',
+  },
+  data: {
+    id: 'dashboard.ExpenditurePage.Batch.data.',
+    defaultMessage: 'Data (max. 400)',
+  },
+  upload: {
+    id: 'dashboard.ExpenditurePage.Batch.upload',
+    defaultMessage: 'Upload .CSV',
+  },
+  importedPayments: {
+    id: 'dashboard.ExpenditurePage.Batch.importedPayments',
+    defaultMessage: 'Imported payments',
+  },
+  value: {
+    id: 'dashboard.ExpenditurePage.Batch.value',
+    defaultMessage: 'Value',
+  },
+  valueWithToken: {
+    id: 'dashboard.ExpenditurePage.Batch.valueWithToken',
+    defaultMessage: '{icon} {token} {amount}',
+  },
+  noTokens: {
+    id: 'dashboard.ExpenditurePage.Batch.noTokens',
+    defaultMessage: 'No tokens',
+  },
+  noPayments: {
+    id: 'dashboard.ExpenditurePage.Batch.noPayments',
+    defaultMessage: 'No payments',
+  },
+  invalidRows: {
+    id: 'dashboard.ExpenditurePage.Batch.invalidRows',
+    defaultMessage: `Not all rows were imported correctly, please review. {button}`,
+  },
+  viewAll: {
+    id: `dashboard.ExpenditurePage.Batch.viewAll`,
+    defaultMessage: 'View all',
+  },
+  tooltipText: {
+    id: `dashboard.ExpenditurePage.Batch.tooltipText`,
+    defaultMessage: `Download and use the .csv import template. Be sure to remove the sample data and view the details are correct once they have been imported.`,
```

Comment:
Sorry, can we please change this to "Download and use the .csv import template. Be sure to remove the sample data and confirm the details are correct once imported."